specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -2919,6 +2919,49 @@
         "activities"
       ]
     },
+    "ValidationActivity": {
```

Comment:
I see that you have added 2 definitions ValidationActivity & ValidationActivityTypeProperties. Where are these definitions used?

Reply:
My understanding is that these definitions are used by our tools to generate the SDKs. I am not aware of any place where these are supposed to be referenced. @hvermis am I missing something?